types/stripe/stripe-tests.ts

```diff
@@ -672,6 +695,11 @@ stripe.accounts.list(
         // asynchronously called
     }
 );
+stripe.accounts.retrieve("acct_17wV8KBoqMA9o2xk").then(
+    function (accounts) {
+        var payouts_enabled = accounts.payouts_enabled;
```

Comment:
If you can change this line to:
`let payouts_enabled: boolean = accounts.payouts_enabled;`

That way it is type checking that `accounts.payouts_enabled` is a boolean.

Reply:
Done